dbt/parser.py

```diff
@@ -92,17 +92,13 @@ def process_refs(flat_graph, current_project):
                 node.get('package_name'))
 
             if target_model is None:
-                dbt.exceptions.ref_target_not_found(
-                    node,
-                    target_model_name,
-                    target_model_package)
-
-            if (dbt.utils.is_enabled(node) and not
-                    dbt.utils.is_enabled(target_model)):
-                if dbt.utils.is_type(node, NodeType.Model):
-                    dbt.exceptions.ref_disabled_dependency(node, target_model)
-                else:
-                    node.get('config', {})['enabled'] = False
```

Comment:
did you mean to remove this? there are two things here, ref target not found (the thing doesn't exist) and ref disabled dependency (target is disabled). i think ref target not found should still be a thing -- did functional testing prove that not to be the case?

Reply:
@cmcarthur yeah -- this code doesn't do anything currently. Disabled nodes are not included in the graph as of https://github.com/fishtown-analytics/dbt/pull/568.

I can leave it in, but I'm pretty sure it's dead code

Reply:
gotcha